package.json

```
@@ -17,7 +17,12 @@
     "build:tools": "tsc -p tsconfig.build.tools.json",
     "clean": "rimraf dist coverage",
     "test": "jest",
-    "lint": "eslint --config .eslintrc.js .",
+    "fix": "yarn fix:eslint && yarn fix:prettier",
+    "fix:eslint": "eslint --config .eslintrc.js --fix .",
+    "fix:prettier": "prettier --write \"{src,test,scripts}/**/**.ts\"",
+    "lint": "yarn lint:eslint && lint:prettier",
```

Just a missing `yarn`.
```suggestion
    "lint": "yarn lint:eslint && yarn lint:prettier",
```